code/controllers/subsystem/profiler.dm

```diff
@@ -1,11 +1,8 @@
-#define PROFILER_FILENAME "profiler.json"
-#define SENDMAPS_FILENAME "sendmaps.json"
-
 SUBSYSTEM_DEF(profiler)
 	name = "Profiler"
 	init_order = INIT_ORDER_PROFILER
 	runlevels = RUNLEVELS_DEFAULT | RUNLEVEL_LOBBY
-	wait = 3000
+	wait = 300 SECONDS
```

Comment:
Why have a big number and not just 5 MINUTES

Reply:
it's the same conversion anyways, the bad thing about `3000` though is that it's not clear it's supposed to be in seconds.

`300 SECONDS` vs `5 MINUTES` is fine though, the conversion is easy to do and it's easy to see what unit the time scale is in.